lib/internal/cluster/master.js

```diff
@@ -36,11 +36,13 @@ let ids = 0;
 let debugPortOffset = 1;
 let initialized = false;
 
-// XXX(bnoordhuis) Fold cluster.schedulingPolicy into cluster.settings?
-let schedulingPolicy = {
+const schedulingPolicies = {
   'none': SCHED_NONE,
   'rr': SCHED_RR
-}[process.env.NODE_CLUSTER_SCHED_POLICY];
+};
+
+// XXX(bnoordhuis) Fold cluster.schedulingPolicy into cluster.settings?
+let { [process.env.NODE_CLUSTER_SCHED_POLICY]: schedulingPolicy } = schedulingPolicies;
```

Comment:
lint-js rule check error is triggered :)

Reply:
Yes, my bad. Done 👍